docs/source/apps/projsync.rst

```diff
@@ -0,0 +1,158 @@
+
+Description
+***********
+
+:program:`projsync` is a program that downloads remote resource files
+into a local directory. This is an alternative to downloading a proj-data-X.Y.Z
+archive file, or using the on-demand :ref:`networking capabilities <network>` of PROJ.
+
+The following control parameters can appear in any order:
+
+.. program:: projsync
+
+.. option:: --endpoint URL
+
+    Defines the URL where to download the master ``files.geojson`` file and then
+    the resource files. Defaults to the value set in :ref:`proj-ini`
+
+.. option:: --local-geojson-file FILENAME
+
+    Defines the filename for the master GeoJSON files that references resources.
+    Defaults to ${endpoint}/files.geojson
+
+.. option:: --user-writable-directory
+
+    Specifies that resource files must be downloaded in the
+    :ref:`user writable directory <user_writable_directory>`. This is the default.
+
+.. option:: --system-directory
+
+    Specifies that resource files must be downloaded in the
+    ${installation_prefix}/share/proj directory. The user launching projsync
+    should make sure it has writing rights in that directory.
+
+.. option:: --target-dir DIRNAME
+
+    Directory into which resource files must be downloaded.
+
+.. option:: --bbox west_long,south_lat,east_long,north_lat
+
+    Specify an area of interest to restrict the resources to download.
+    The area of interest is specified as a
+    bounding box with geographic coordinates, expressed in degrees in a
+    unspecified geographic CRS.
+    `west_long` and `east_long` should be in the [-180,180] range, and
+    `south_lat` and `north_lat` in the [-90,90]. `west_long` is generally lower than
+    `east_long`, except in the case where the area of interest crosses the antimeridian.
+
+.. option:: --spatial-test contains|intersects
+
+    Specify how the extent of the resource files
+    are compared to the area of use specified explicitly with :option:`--bbox`.
+    By default, any resource files whose extent intersects the value specified
+    by :option:`--bbox` will be selected.
+    If using the ``contains`` strategy, only resource files whose extent is
+    contained in the value specified by :option:`--bbox` will be selected.
+
+.. option:: --source-id ID
+
+    Restrict resource files to be downloaded to those whose source_id property
+    contains the ID value. Specifying ``?`` as ID will list all possible values.
+
+.. option:: --area-of-use NAME
+
+    Restrict resource files to be downloaded to those whose area_of_use property
+    contains the NAME value. Specifying ``?`` as NAME will list all possible values.
+
+.. option:: --file NAME
+
+    Restrict resource files to be downloaded to those whose name property
+    contains the NAME value. Specifying ``?`` as NAME will list all possible values.
+
+.. option:: --all
+
+    Ask to download all files.
+
+.. option:: --exclude-world-coverage
+
+    Exclude files which have world coverage.
```

Comment:
What box defines world coverage?

Reply:
"grid_e - grid_w > 359 && grid_n - grid_s > 179", to exclude EGM files